packages/in_app_purchase/in_app_purchase_storekit/darwin/in_app_purchase_storekit/Sources/in_app_purchase_storekit/StoreKit2/InAppPurchasePlugin+StoreKit2.swift

```diff
@@ -146,6 +145,31 @@ extension InAppPurchasePlugin: InAppPurchase2API {
     }
   }
 
+  /// Wrapper method around StoreKit2's countryCode() method
+  /// https://developer.apple.com/documentation/storekit/storefront/countrycode
+  func countryCode(completion: @escaping (Result<String, Error>) -> Void) {
+    Task {
+      guard let currentStorefront = await Storefront.current else {
+        let error = PigeonError(
+          code: "storekit2_failed_to_fetch_country_code",
+          message: "Storekit has failed to fetch the country code.",
+          details: "Storekit has failed to fetch the country code.")
+        return completion(.failure(error))
+      }
+      return completion(.success(currentStorefront.countryCode))
```

Comment:
this is a weird way to write it - it's returning whatever to be returned from `completion` closure, which is `Void`.